LeaderboardBackend/Services/IRunService.cs

```diff
@@ -11,10 +11,45 @@ public interface IRunService
     Task<Run?> GetRun(Guid id);
     Task<GetRunsForCategoryResult> GetRunsForCategory(long id, Page page, bool includeDeleted = false);
     Task<CreateRunResult> CreateRun(User user, Category category, CreateRunRequest request);
+    /// <returns>
+    ///     One of four cases (none of which return any inner data):
+    ///     <list type="bullet">
+    ///         <item>
+    ///             <term>BadRole</term>
+    ///             <description>
+    ///                 The user is either not a
+    ///                 <see cref="UserRole.Administrator"/>, or they are a
+    ///                 <see cref="UserRole.Confirmed" /> but are trying to
+    ///                 edit a run that isn't theirs.
```

Comment:
These should actually be two different results. Three if you count a non-admin user trying to edit a deleted run.

Reply:
This relates to https://github.com/leaderboardsgg/leaderboard-backend/pull/307#issuecomment-2820868130 imo. I can only imagine disambiguation would be useful only to power users/API clients. If we do want to tell clients why they've 403'd, then yeah sure I can split the results out (and update the docs of course), on top of adding reasons to the endpoint response's title (detail?) field.